tests/decorators/tests.py

```diff
@@ -492,7 +493,7 @@ def test_decorators_syanc_and_async_capable(self):
                 self.assertTrue(decorator.async_capable)
 
 
-class XFrameOptionsDecoratorsTests(TestCase):
+class XFrameOptionsDecoratorsTests(SimpleTestCase):
```

Comment:
This has been changed from `TestCase` to `SimpleTestCase` for two reasons:

1. `TestCase` doesn't seem to await async functions properly. I'm not sure exactly why, but this doesn't seem like the right PR o fix it in.
2. As per the [Django docs](https://docs.djangoproject.com/en/dev/topics/testing/tools/#testcase) since we're not using a database for any of these tests, we should be using `SimpleTestCase` instead of `TestCase`.